### ISSUE TITLE
Keine Statistik gefunden. arcgis API Token Required

### ISSUE THREAD

**Issue by FlavourDave92** [marcusraitner/COVID-19-Dashboard#124]
Hey, 

kann es sein, dass das Script nicht mehr funktioniert, weil die arcgis API (jetzt?) ein Token benötigt?

Bekomme beim Aufrufen jetzt immer „Keine Statistik gefunden.“ angezeigt, der Debug Log sagt:

```json
{"error":{"code":499,"message":"Token Required","messageCode":"GWM_0003","details":["Token Required"]}}
```

Gibt es eine Möglichkeit das zu umgehen?

Danke für die tolle Arbeit und das tolle Script!

**Comment by marcusraitner** [comment]
Ja, kann sein. Siehe auch https://github.com/marcusraitner/COVID-19-Dashboard/issues/122 Du solltest einstweilen `frozen=y` setzen
